wiki/Upgrading-Target-Sdk-Guide.md

```diff
@@ -0,0 +1,148 @@
+- In some cases, a real device may need to be used instead of an emulator since certain features change behavior on an emulator (such as the drag and drop interaction).
+- If the app isn't already using the new version of Android as its compile SDK version, then it may fail to build. Any build failures should be filed as issues on the issue tracker and fixed before this part of the process can continue.
+
+The local version of the app should be deployed to an emulator (or a real device if one is available) and tested as such:
+- Any user behaviors that tie to the areas of concern noted in the tracking issue from part (2) should be explicitly tested to ensure they still behave correctly.
+  - Note that in some cases this may require using both a handset and tablet emulator configuration if there are tablet-specific or layout-specific areas identified.
+  - Note that #5137 may provide some good context on how to test certain types of changes that may not be as simple as manually performing a certain user action and may instead require a clever code change.
+- A general analysis should be peformed by testing the following scenarios:
+  - Profile creation/deletion and login.
+  - Playing, pausing, resuming, and finishing a lesson.
+  - Ensuring all interactions work (play through every test topic prototype exploration).
+  - Ensuring LaTeX and in-lesson images load correctly.
+  - Ensuring that lesson progress correctly saves per profile.
+  - Ensuring that profile avatars can be correctly set.
+  - Ensuring that hints and solutions work correctly.
+  - Ensuring that wrong answers are handled correctly.
+  - Checking that concept and revision cards work correctly.
+  - Verifying that events are logged (either per Firebase analytics if you have access, or the developer options menu using a build of ``//:oppia_dev``--note that ``oppia_dev`` should only be used for this specific verification and not any of the others).
+
+Any breakages should be noted, and then checked against a version of the app without the target SDK. From there:
+- If the breakage still occurs, file a new bug noting the problem and mention in the 'additional context' section that it was found during target SDK testing but was determined as unrelated.
+- If the breakage does not occur on the non-upgraded version of the app, file a new bug in the issue tracker and mention that it's specific to the new target SDK version and is a blocking issue.
+
+Please note all found blocking issues with their issue numbers in a follow-up reply to the tracking issue. Any other thoughts or findings during testing can also be noted in the tracking issue (similar to the comments in #5137). Please also note in the tracking issue when testing has concluded.
+
+### Part 4: Fixing and stabilizing support
+
+Work on fixing all identified problems from parts (3) and (4) (either by directly fixing the problems via code changes, or via coordination with other members of the team).
```

Comment:
Since this is a very short section, I'm not sure the subheadings really add much value. Do you find the current section process difficult to follow such that additional organizing might help?

Reply:
Yeah, makes sense. The current version by itself is easy to follow, so I'll resolve it. Thanks!